visualization/utils/strategies/base_strategy.py

```diff
@@ -0,0 +1,246 @@
+import logging
+from dataclasses import asdict
+from typing import Callable, Dict, Iterable, List, Tuple, Union
+
+import dash_bootstrap_components as dbc
+from dash import html
+from flask import current_app
+from layouts import get_input_group_layout, get_single_prompt_output_layout, get_switch_layout, get_text_area_layout
+from utils.common import examples
+
+from nemo_skills.code_execution.sandbox import get_sandbox
+from nemo_skills.inference.generate_solutions import InferenceConfig
+from nemo_skills.inference.prompt.utils import PromptConfig, context_templates, get_prompt
+from nemo_skills.inference.server.model import get_model
+from visualization.settings.constants import QUERY_INPUT_TYPE
+
+
+class ModeStrategies:
+    def __init__(self):
+        self.sandbox = None
+        self.config = current_app.config['prompt_explorer']
+
+    def sandbox_init(self, utils):
+        if self.sandbox is None:
+            self.sandbox = get_sandbox(
+                sandbox_type=self.config['sandbox']['sandbox_type'],
+                host=self.config['server']['host'],
```

Comment:
why don't you reuse the config files information here? The sandbox does not have to be on the same host + in the future we'd have more sandboxes with different parameters, so better to have generic init here